src/org/parosproxy/paros/extension/option/OptionsJvmPanel.java

```diff
@@ -93,8 +109,41 @@ private ZapTextField getJvmOptionsField() {
 		return jvmOptionsField;
 	}
 	
+	private JLabel getSizeMemoryLabel() {
+		if (sizeMemoryLabel == null) {
+			sizeMemoryLabel = new JLabel();
+			updateMemoryLabel(sizeMemoryLabel, "jvm.options.memory.size", Runtime.getRuntime().totalMemory());

```

Comment:
I guess we could remove these, `initParam` is always called before showing the panel, wdyt?

Reply:
Sure. I’ll remove the updates from the getters and put a comment leading the updates in init so that it’s obvious for the next guy 🙂

Reply:
btw, does it worth using `ZapLabel`s instead? (to allow select and copy)

Reply:
Sure, will tweak that too.